src/components/Icons/Icons.stories.tsx

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import { Story, Meta } from '@storybook/react/types-6-0';
+import { IconGitHub } from './index';
+
+export default {
+  title: 'Developer DAO/00-components/Icons',
```

Comment:
Similar to above comment about flattening but I'd be okay with grouping this into a folder since we have a bunch of icons already, so I think that makes sense.

```suggestion
  title: 'icons/Icons',
```

Reply:
OH WAIT! This is a single story for all of the icons. Yeah let's just make it `title: 'Icons'`